Exercise_2.cpp

```diff
@@ -19,21 +20,43 @@ StackNode* newNode(int data)
 int isEmpty(StackNode* root) 
 { 
     //Your code here 
+    if(root == NULL)
+        return true;
+    return false;
+
 } 
   
 void push(StackNode** root, int data) 
 { 
     //Your code here 
+    StackNode* node = newNode(data);
+    node->next = *root;
+    *root = node;
 } 
-  
+
+//Why does pop method need a parameter since Stack is LIFO?  

```

Comment:
Can you actually do this without having access to root ?

Reply:
That makes sense. I was a little confused initially. Thank you for the response